axonserver/src/main/java/io/axoniq/axonserver/component/version/ClientVersionUpdate.java

```diff
@@ -0,0 +1,31 @@
+package io.axoniq.axonserver.component.version;
```

Comment:
Missing javadoc overall in this class, on top level, the constructors and the methods.